Examples/Python/tests/conftest.py

```diff
@@ -390,16 +389,20 @@ def _factory(s):
 def _do_material_recording(d: Path):
     from material_recording import runMaterialRecording
 
+    s = acts.examples.Sequencer(events=2, numThreads=1)
+
     detector, trackingGeometry, decorators = getOpenDataDetector()
 
-    detectorConstructionFactory = (
-        acts.examples.geant4.dd4hep.DDG4DetectorConstructionFactory(detector)
-    )
+    with detector:
```

Comment:
This is a bit awkward in that the detector variable exists as empty afterwards. I liked the implied lifetime of the previous approach, but I see that this way it's easier to generalize to all detector types. 

Anyway, not critical.